emissionsapi/openapi.yml

```diff
@@ -207,3 +247,39 @@ components:
           end:
             type: date-time
             example: 2019-02-10T13:03:40.254000Z
+    Statistics:
+      type: array
+      description: Array of statistical data about carbon monoxide for specified intervals.
+      items:
+        type: object
+        properties:
+          value:
```

Comment:
I am curious. Why do you call it `value`? I would have go with something like `statistics` or something else less generic. Also should you use singular or plural here?

Reply:
Think REST. You are already in the statistics entity. Naming these statistics would be weird. Would that be the statistics of the statistics?

For the `value` vs `values`, the other key is also named `time` and not `times`. But then, both contain just a single object, not an array. Hence, you get something like `statistics.value.min` or `statistics.time.interval_start`